endstate_correction/tests/test_sampling.py

```diff
@@ -36,14 +34,11 @@ def test_sampling():
         f"{hipen_testsystem}/{system_name}/{system_name}.str",
     )
     # define region that should be treated with the qml
-    chains = list(psf.topology.chains())
-    ml_atoms = [atom.index for atom in chains[0].atoms()]
-
-    sim = create_charmm_system(
-        psf=psf, parameters=params, env="vacuum", ml_atoms=ml_atoms
-    )
+    sim = setup_vacuum_simulation(psf, params)
```

Comment:
I think it might be good to separate the code that is not relevant to the switching to NpT traj to another PR>

Reply:
I have changed the target branch, I hope this is now easier to separate.
Sorry for that!